src/mkdocs_llmstxt/_internal/debug.py

```diff
@@ -79,7 +79,10 @@ def _get_debug_info() -> _Environment:
     """
     py_name, py_version = _interpreter_name_version()
     packages = ["mkdocs-llmstxt"]
-    variables = ["PYTHONPATH", *[var for var in os.environ if var.startswith("MKDOCS_LLMSTXT")]]
+    variables = [
+        "PYTHONPATH",
+        *[var for var in os.environ if var.startswith("MKDOCS_LLMSTXT")],
+    ]
```

Comment:
chore: I see IDE auto-formatting here, we use a 120 line-length, should be reverted :smile: Ruff's config file is in the `config` folder. We provide a `make vscode` command that will configure VSCode accordingly, if that's the IDE you're using. **It will overwrite the current project config.**

Reply:
I think I just ran `ruff format/check` on `src` assuming the configuration would live in the `pyproject.toml`, reverted the changes.